plugins/pci-mei/fu-mei-common.c

```diff
@@ -49,26 +49,45 @@ fu_mei_common_cmp_version(FuMeiVersion *vers1, FuMeiVersion *vers2)
 FuMeiIssue
 fu_mei_common_is_csme_vulnerable(FuMeiVersion *vers)
 {
-	if (vers->major == 11 && (vers->minor == 8 || vers->minor == 11 || vers->minor == 22)) {
-		return vers->hotfix >= 70 ? FU_MEI_ISSUE_PATCHED : FU_MEI_ISSUE_VULNERABLE;
-	} else if (vers->major == 12 && vers->minor == 0) {
-		return (vers->hotfix == 49 || vers->hotfix >= 56) ? FU_MEI_ISSUE_PATCHED
-								  : FU_MEI_ISSUE_VULNERABLE;
-	} else if (vers->major == 13 && vers->minor == 0) {
-		return vers->hotfix >= 21 ? FU_MEI_ISSUE_PATCHED : FU_MEI_ISSUE_VULNERABLE;
-	} else if (vers->major == 14 && vers->minor == 0) {
-		return vers->hotfix >= 11 ? FU_MEI_ISSUE_PATCHED : FU_MEI_ISSUE_VULNERABLE;
+	struct {
+		guint8 major_eq;
+		guint8 minor_eq;
+		guint8 hotfix_ge;
+	} verdata[] = {{11, 8, 92},
+		       {11, 12, 92},
```

Comment:
Shouldn't this be 11, 11?

Reply:
I think it was *intel's* typo, the python code they use is:

    if vers[0] == 11 and vers[1] == 8 and vers[2] >= 92:
        return glob.DISCOVERY_NOT_VULNERABLE_PATCHED
    if vers[0] == 11 and vers[1] == 12 and vers[2] >= 92:
        return glob.DISCOVERY_NOT_VULNERABLE_PATCHED
    if vers[0] == 11 and vers[1] == 22 and vers[2] >= 92:
        return glob.DISCOVERY_NOT_VULNERABLE_PATCHED
    if vers[0] == 12 and vers[1] == 0 and vers[2] >= 90:
        return glob.DISCOVERY_NOT_VULNERABLE_PATCHED
    if vers[0] == 13 and vers[1] == 0 and vers[2] >= 60:
        return glob.DISCOVERY_NOT_VULNERABLE_PATCHED
    if vers[0] == 13 and vers[1] == 30 and vers[2] >= 30:
        return glob.DISCOVERY_NOT_VULNERABLE_PATCHED
    if vers[0] == 13 and vers[1] == 50 and vers[2] >= 20:
        return glob.DISCOVERY_NOT_VULNERABLE_PATCHED
    if vers[0] == 14 and vers[1] == 1 and vers[2] >= 65:
        return glob.DISCOVERY_NOT_VULNERABLE_PATCHED
    if vers[0] == 14 and vers[1] == 5 and vers[2] >= 45:
        return glob.DISCOVERY_NOT_VULNERABLE_PATCHED
    if vers[0] == 15 and vers[1] == 0 and vers[2] >= 40:
        return glob.DISCOVERY_NOT_VULNERABLE_PATCHED
    if vers[0] == 15 and vers[1] == 40 and vers[2] >= 20:
        return glob.DISCOVERY_NOT_VULNERABLE_PATCHED
    if vers[0] == 16:
        return glob.NOTVULNERABLE
    return glob.DISCOVERY_VULNERABLE